File: app/src/main/java/ie/dublinbuspal/android/view/favourite/FavouritesModel.java
package ie.dublinbuspal.android.view.favourite;

import java.util.List;

import ie.dublinbuspal.model.favourite.FavouriteStop;

public interface FavouritesModel {

    void setFavourites(List<FavouriteStop> favourites);

    List<FavouriteStop> getFavourites();

}

File: app/src/main/kotlin/ie/dublinbuspal/android/view/favourites/FavouritesModel.kt
package ie.dublinbuspal.android.view.favourites

import ie.dublinbuspal.model.favourite.FavouriteStop

interface FavouritesModel {

    fun setFavourites(favourites: List<FavouriteStop>)

    fun getFavourites(): List<FavouriteStop>

}
